### PR TITLE
Ev/docs

### PR DESCRIPTION
**Pull request by twakes**
New docs updated with Ev's changes and Taylor's edits:
* Updated quickstart to work with OSS distribution.
* Made it clear Ops Center is only available in the Enterprise Version.
* Updated Overview to be more current.

### CODE REVIEW

File: docs/5.x/opscenter.md [hunk 1]
```diff
@@ -1,73 +1,70 @@
-# Setting up an Ops Center
+# Introduction
 
-An Ops Center controls access and lifecycle of Applicaion Clusters and provides a distribution endpoint
-for Application Bundles to be installed on Gravity Clusters.
+The Gravity Ops Center is a multi-cluster control plane available in the Enterprise version of Gravity. It reducues the operational overhead of managing multiple Gravity Clusters and Applications Bundles by allowing users to:
```
Comment (r0mant) [inline docs/5.x/opscenter.md]: reducues -> reduces

File: docs/5.x/opscenter.md [hunk 2]
```diff
@@ -1,73 +1,70 @@
...
 
-## Pre-requisites
+* Publish Application Bundles and manage their versions.
```
Comment (r0mant) [inline docs/5.x/opscenter.md]: Are we still calling this "Application Bundles", not "Cluster Images"? Will that terminology change come later?

File: docs/5.x.yaml [hunk 3]
```diff
@@ -32,7 +32,7 @@ pages:
     - Cluster Monitoring: monitoring.md
     - Remote Management: manage.md
     - Ops Center: opscenter.md
-    - Application Catalog: catalog.md
+    - hidden: catalog.md
```
Comment (r0mant) [inline docs/5.x.yaml]: Why has this become hidden?

File: docs/5.x/overview.md [hunk 4]
```diff
@@ -1,76 +1,89 @@
...
+!!! tip "Gravity Roadmap Tip":
+    Kubernetes community is working on finalizing the cluster API spec. Once it
+    becomes production ready, Gravity developers will be looking into adopting the
+    future standard to replace the application manifest in the future. Meanwhile, 
+    it continues to be the only production-ready method of describing hardware
+    requirements for K8s clusters.
 
-To prepare a Kubernetes application for distribution via Gravity, you have to write
-an Application Manifest. Below is a sample of a simple Application Manifest in YAML format. It follows Kubernetes configuration conventions:
+Below is a simple Application Manifest in YAML format. It follows Kubernetes
```
Comment (r0mant) [inline docs/5.x/overview.md]: I think it meant to say "sample", not "simple". The manifest below is not simple at all as it include all possible fields :)

File: docs/5.x/quickstart.md [hunk 5]
```diff
@@ -1,231 +1,373 @@
...
+* [5/6] Generating the cluster snapshot
+* [6/6] Saving the snapshot as mattermost.tar
+* [6/6] Build completed in 2 minutes 
+```
 
-```bsh
-$ tele keys new
+Let's review what just happened. `tele build` did the following:
+
+* Downloaded Kubernetes binaries and Gravity tooling from `s3://hub.gravitational.io`. The application "runtime".
```
Comment (r0mant) [inline docs/5.x/quickstart.md]: `The application "runtime".` sentence seems a bit out of place. If it's supposed to clarify the previous sentence, I would either write it after a dash (Downloaded blah-blah - the application "runtime") or inside parentheses.

File: docs/5.x/quickstart.md [hunk 6]
```diff
@@ -1,231 +1,373 @@
...
+Sat Jan 12 05:35:43 UTC Install system application site:5.2.4
+Sat Jan 12 05:36:59 UTC Install system application kubernetes:5.2.4
+Sat Jan 12 05:37:00 UTC Install application mattermost:2.2.0
+Sat Jan 12 05:37:10 UTC Enable elections
+Sat Jan 12 05:37:12 UTC Operation has completed
+Sat Jan 12 05:37:13 UTC Installation succeeded in 6m3.257480586s
+```
 
-### Online Installer
+**Congratulations!** You have created a fully functional, highly available
```
Comment (r0mant) [inline docs/5.x/quickstart.md]: The command above installed a single-node cluster so I would remove "highly available".

File: docs/5.x/opscenter.md (hunk 2 above)
Comment (twakes) [reply docs/5.x/opscenter.md]: Yes, I think that is next update.

File: docs/5.x.yaml (hunk 3 above)
Comment (twakes) [reply docs/5.x.yaml]: I didn't make this change. Perhaps @kontsevoy knows.

**Pull request merged**